### DISCUSSION TITLE
How to change the root fontsize

### DISCUSSION THREAD

**Discussion by zeakey** [tusen-ai/naive-ui#6303] [Q&A]
The root font size is defaulted to 14px, which is pretty small. How can I change this value?

I tried to insert the following codes into my App.vue.
```
const themeOverrides: GlobalThemeOverrides = {
  common: {
    fontSize: '16px'
  }
}


<template>
  <n-config-provider :theme="globalTheme" :locale="uiLocal" :date-locale="uiDateLocal" :theme-overrides="themeOverrides">
    <n-global-style />
    <naive-provider>
      <router-view />
    </naive-provider>
  </n-config-provider>
</template>
```

but it seems not to be working and I still get body font-size 14px
![image](https://github.com/user-attachments/assets/01bac687-6b53-4a1e-87b8-f3d2f659f223)

**Comment by jahnli** [comment]
theme fontSize
